coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/hierarchy/ProposalHierarchyServiceImpl.java

```diff
@@ -736,42 +740,48 @@ protected void syncAllPersonnelAttachments(DevelopmentProposal hierarchyProposal
+                // people that exist on the parent, ignore those bios.
+                if (!isBioInNewChildDuplicate(isNewChild, hierarchyProposal, srcPropPersonBio)
+                        && !personInMultipleProposals(srcPropPersonBio.getPersonId(), hierarchyProposal)) {
+                        // mark those persons that are not in multiple proposals
+                        // and remove this persons bio from parent
+                        // since they will be copied over again later. We need to do this so if a bio is updated at the
+                        // child it will sync up.
+                        personInMultipleProp.put(srcPropPersonBio.getPersonId(), false);
+                        for (Iterator<ProposalPersonBiography> iteratorParent = hierarchyProposal.getPropPersonBios().iterator(); iteratorParent.hasNext(); ) {
+                            ProposalPersonBiography parentPropPersonBio = iteratorParent.next();
+                            if (StringUtils.equals(srcPropPersonBio.getPersonId(), parentPropPersonBio.getPersonId())) {
+                                iteratorParent.remove();
+                            }
+                        }
                 }
             }
-        }
-
-        // go over the child bios list and if person is not in multiple proposals and the
-        // bio does not exist for the person in the parent, add that bio
-        for (ProposalPersonBiography srcPropPersonBio : childProposal.getPropPersonBios()) {
-            if (personInMultipleProp.get(srcPropPersonBio.getPersonId()) != null &&
-                    !personInMultipleProp.get(srcPropPersonBio.getPersonId()) &&
-                    !isDuplicateBio(srcPropPersonBio, hierarchyProposal)) {
-                ProposalPersonBiography destPropPersonBio;
-                destPropPersonBio = (ProposalPersonBiography) deepCopy(srcPropPersonBio);
-                destPropPersonBio.setDevelopmentProposal(hierarchyProposal);
-                destPropPersonBio.setProposalNumber(hierarchyProposal.getProposalNumber());
-                destPropPersonBio.setProposalPersonNumber(getProposalPersonNumber(destPropPersonBio.getPersonId(), hierarchyProposal));
-                destPropPersonBio.setVersionNumber(0L);
-                newList.add(destPropPersonBio);
 
+            // go over the child bios list and if person is not in multiple proposals add it
+            for (ProposalPersonBiography srcPropPersonBio : childProposal.getPropPersonBios()) {
+                // if the proposal is JUST being linked to a hierarchy and if this proposal has bios for
+                // people that exist on the parent, ignore those bios.
+                if (!isBioInNewChildDuplicate(isNewChild, hierarchyProposal, srcPropPersonBio)
+                    && personInMultipleProp.get(srcPropPersonBio.getPersonId()) != null &&
+                    !personInMultipleProp.get(srcPropPersonBio.getPersonId())) {
```

Comment:
Yeah, when I initially added it, I meant to use it for more than just a check but never ended up using it that way. I would rather not change this now though if it is minor. This code change itself I think risky at this point and if more hierarchy personnel attachments issues are filed, I am going to revert.